.eslintrc.js

```diff
@@ -4,10 +4,7 @@ module.exports = {
     ecmaVersion: 2017,
     sourceType: 'module'
   },
-  extends: [
-    'eslint:recommended',
-    'plugin:ember-suave/base'
-  ],
```

Comment:
I actually removed it since the build was failing. Ill try adding https://github.com/DockYard/eslint-plugin-ember-suave to the dependencies to see if that fixes it

Reply:
I tried adding the ember-suave dependency but now theres a ton of lint failures. Should I make a separate PR for that?

Reply:
Did you add just `plugin:ember-suave/base`? IIRC there is `ember-suave/base` which is less restrictive. Then there is: `ember-suave/recommended` that is a lot more strict. If you're using base and it's still failing, let's just merge this in now and work on another PR later. I don't want to get too bogged down by this one.

Reply:
good call looks like base works. Pushing now